sktime/forecasting/neuralforecast.py

```diff
@@ -96,6 +96,10 @@ class NeuralForecastRNN(_NeuralForecastAdapter):
         whether ``TimeSeriesDataLoader`` drops last non-full batch
     trainer_kwargs : dict (default=None)
         keyword trainer arguments inherited from PyTorch Lighning's trainer [6]_
+    optimizer : pytorch optimizer (default=None)
+        optimizer to use for training, if passed with None defaults to Adam
```

Comment:
Can you please add supported iptimisers and a link tobthe documentation in reference section? 

Same comment applies for other models too.